lib/slack-ruby-bot/rspec/support/slack-ruby-bot/respond_with_slack_message.rb

```diff
@@ -1,27 +1,32 @@
 require 'rspec/expectations'
-
 RSpec::Matchers.define :respond_with_slack_message do |expected|
+  include SlackRubyBot::SpecHelpers
   match do |actual|
-    client = if respond_to?(:client)
-               send(:client)
-             else
-               SlackRubyBot::Client.new
-             end
+    client = respond_to?(:client) ? send(:client) : SlackRubyBot::Client.new
+    def client.test_messages
```

Comment:
Try `client.attr_accessor :test_messages` instead?

Reply:
No go, that causes an error since it's an instance of a class, rather than a class definition.